fda-model/alternatives/Alternative.cs

```diff
@@ -7,23 +7,60 @@ public class Alternative
     {
         private Scenario _currentYear;
         private Scenario _futureYear;
+        private Int64 _periodOfAnalysis;
+
         //probably need getters and setters
-        public Alternative(Scenario currentYear, Scenario futureYear){
+        public Alternative(Scenario currentYear, Scenario futureYear, Int64 periodOfAnalysis){
             _currentYear = currentYear;
             _futureYear = futureYear;
+            _periodOfAnalysis = periodOfAnalysis;
         }
         public IList<metrics.IContainResults> ComputeEEAD(interfaces.IProvideRandomNumbers rp, Int64 iterations, double discountRate){
             //probably instantiate a rng to seed each impact area differently
-            IList<metrics.IContainResults> currentEAD = _currentYear.Compute(rp,iterations);
-            IList<metrics.IContainResults> futureEAD = _futureYear.Compute(rp, iterations);
-            //discoiunt future ead
-            IList<metrics.IContainResults> discountedEAD = discount(futureEAD,discountRate, _futureYear.Year - _currentYear.Year);
-            //combine the future and current ead
-            return currentEAD;//not right, fix this.
+            IList<metrics.IContainResults> baseEAD = _currentYear.Compute(rp,iterations);
```

Comment:
@HenryGeorgist could you please point out to me how I can access baseEAD and mlfEAD? I do not see any available properties or methods that can get a double from the IList<metrics.IContainResults> object.

Reply:
I'd like to get some unit testing running on this code asap.

Reply:
sure for reference the test here shows it in the other simulation tests:
https://github.com/HydrologicEngineeringCenter/fda-model/blob/b968c90ccff9c71400758caceb74a03c7f3054e4/fda-model-test/simulationShould.cs#L42

it should be something like baseEAD.MeanEAD("Total");